core/geometry/src/polyface/PolyfaceClip.ts

```diff
@@ -47,15 +47,7 @@ class ClipCandidate {
 }
 
 /**
- * A pair of PolyfaceBuilder objects, for use by clippers that emit inside and outside parts.
- * * There are nominally 4 builders:
```

Comment:
Any specific reason for removing all these doc details?!

Reply:
They were incorrect.  Probably copied from native, where the implementation is different.